locale/en-US/messages.properties

```diff
@@ -1 +1,8 @@
-config.labelFromProperties=Label defined in a properties file
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+config.rule.item.controls.edit.label=✏
```

Comment:
Yes. I used emoji characters instead of icon images. On my Ubuntu 14.04 it is shown as I expected, but it will be just a square if your screen font doesn't have a glyph for the character code. Should I use icon image instead of character?